dvc/config.py

```diff
@@ -108,6 +108,7 @@ class RelPath(str):
         Optional("interactive", default=False): Bool,
         Optional("analytics", default=True): Bool,
         Optional("hardlink_lock", default=False): Bool,
+        Optional("no-scm", default=False): Bool,
```

Comment:
```suggestion
        Optional("no_scm", default=False): Bool,
```
to keep it consistent with the rest.